src/common/components/Header/index.js

```diff
@@ -1,21 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { withRouter } from 'react-router-dom';
 
-import { Layout, Navbar } from '../';
+import ResponsiveNavMenu from '../../../packages/responsive-nav-menu';
 
-const Header = ({ children, history, location }) => (
+import { Layout } from '../';
+
+const Header = ({ children }) => (
   <Layout.Header>
-    <Navbar activePathname={location.pathname} history={history}>
-      {children}
-    </Navbar>
+    <ResponsiveNavMenu>{children}</ResponsiveNavMenu>
```

Comment:
would rename to `ResponsiveNavigation` what do you think? Menu is synonym to navigation i guess so no need for NavMenu